static/app-strings.json

```diff
@@ -518,7 +518,7 @@
   "Terms of Service": "Terms of Service",
   "Learn More": "Learn More",
   "Learn more": "Learn more",
-  "Community": "Community",
+  "Community": "Fellowship",
```

Comment:
Not sure if we want to change the wording, but I'll leave that for others to debate.

But the main issue here is that the file is not meant to be edited that way (hard to sync the meaning with other languages).  
- [ ] I think we can revert and ignore `app-string.json` for this PR.  We'll update it post-PR.

Reply:
Discussed in Slack; changes understood (only doing it for English).